ui/pages/build-status.js

```diff
@@ -265,13 +300,62 @@ const enhanceWithCoverageData = async reposByArea => {
   }, sumEnhanced);
 };
 
+const repoQuery = gql`
+  query getPipeline($project: String!) {
```

Comment:
Q: Isn't `login` just as good a name?

Reply:
Right, my head was more focused on the different "projects" (`cfengine` & `mender`) - but I guess that is not quite right in some cases... - will adjust!